.gitignore

```diff
@@ -1,10 +1,160 @@
-conanfile.txt
-conanbuildinfo.txt
-conanbuildinfo.args
```

Comment:
Please don't remove these from the gitignore as they are generated by the build.